ui/app/templates/components/configure-ssh-secret.hbs

```diff
@@ -31,14 +31,14 @@
       <ConfirmAction
         @buttonClasses="button"
         @confirmMessage="This will remove the CA certificate information."
-        @onConfirmAction={{action "saveConfig" (hash delete=true)}}
+        @onConfirmAction={{this.delete}}
```

Comment:
I considered calling `@saveConfig` directly here and on line 41 but decided against because the parent isn't expecting a form event and I wanted to avoid making any changes higher up that could potentially introduce regressions